src/pattern-library/components/patterns/UsingComponentsPage.tsx

```diff
@@ -166,6 +171,37 @@ export default function UsingComponentsPage() {
         </Library.Pattern>
       </Library.Section>
 
+      <Library.Section
+        title="Transition components"
+        intro={
+          <>
+            <p>
+              <strong>Transition components</strong> have a common props API to
+              represent their visibility and allows to hook some callbacks for
+              when it changes.
+            </p>
+            <p>
+              Example: <code>Slider</code>.
+            </p>
+          </>
+        }
+      >
+        <Library.Pattern title="Props API">
+          <p>
+            <strong>Transition components</strong> take these common props
+            unless documented otherwise:
```

Comment:
I'm not completely sure if the `unless documented otherwise` part applies here. I left it because it is never technically wrong, but I have mixed feelings.